project/HMDABuild.scala

```diff
@@ -37,6 +35,8 @@ object HMDABuild extends Build {
 
   val httpDeps = akkaDeps ++ Seq(akkaHttp, akkaHttpJson, akkaHttpTestkit)
 
+  val scalazDeps = scalaz
```

Comment:
Seems more consistent to have this value be `Seq(scalaz)` (with a corresponding removal on line 78).